Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -696,16 +696,21 @@ class BrowserViewController: UIViewController {
         homePanelIsInline = inline
 
         if homePanelController == nil {
-            homePanelController = HomePanelViewController()
-            homePanelController!.profile = profile
-            homePanelController!.delegate = self
-            homePanelController!.appStateDelegate = self
-            homePanelController!.url = tabManager.selectedTab?.displayURL
-            homePanelController!.view.alpha = 0
+            self.homePanelController = HomePanelViewController()
+            if let homePanelController = self.homePanelController {
+                homePanelController.profile = profile
+                homePanelController.delegate = self
+                homePanelController.appStateDelegate = self
+                homePanelController.url = tabManager.selectedTab?.displayURL
+                homePanelController.view.alpha = 0
 
-            addChildViewController(homePanelController!)
-            view.addSubview(homePanelController!.view)
-            homePanelController!.didMoveToParentViewController(self)
+                addChildViewController(homePanelController)
+                view.addSubview(homePanelController.view)
+                homePanelController.didMoveToParentViewController(self)
+            }
+        }
+        guard let homePanelController = self.homePanelController else {
+            return
```

Comment:
Perhaps throw an `assertionFailure` in here since This Should Never Happen (TM).